test/core.js

```diff
@@ -54,6 +54,14 @@ describe('Core Test', function() {
     expect(apiVersion.minor).to.be.a('number');
   });
 
+  it('getAvailableDevices should be a function', () => {
+    expect(core.getAvailableDevices).to.be.a('function');
+  })
+
+  it('getAvailableDevices should return a array', () => {
+    expect(core.getAvailableDevices()).to.be.a('array');
+  })
+
```

Comment:
Please add test cases for
1. valid output, like it should always contain CPU.
2. invalid input, it should raise an Error

Reply:
Done